src/Equinor.ProCoSys.IPO.Infrastructure/Migrations/20210319152139_MakeIposWithMcPkgScopeDP.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Equinor.ProCoSys.IPO.Infrastructure.Migrations
+{
+    public partial class MakeIposWithMcPkgScopeDP : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql(
+                "UPDATE INVITATIONS SET TYPE = 0 FROM INVITATIONS JOIN MCPKGS MC ON MC.InvitationId = INVITATIONS.Id WHERE TYPE = 1");
```

Comment:
@janid1967 do you think this is okay?

Will confirm with Kristen than we can make all MDPs with mc scope to DPs (since this will be the new rule)

Reply:
1) Have you tried the update in DB? I would have written 
UPDATE INVITATIONS SET TYPE = 0 WHERE ID IN (SELECT ID FROM  FROM INVITATIONS JOIN MCPKGS MC ON MC.InvitationId = INVITATIONS.Id WHERE TYPE = 1)
2) If I understand the business correct, this can (will?) lead to having DP's with both mcpkgs and commpgs in prod, which is from now on illegal. Will handling/editing/signing such work ok?

Reply:
1. I have run this script in my sandbox, so I have checked that it works.

2. This will not lead to DPs having both mc and comm pkg scope. This was never allowed. This is to change all MDPs with mc pkg scope to DPs.